config/base.edn

```diff
@@ -1,4 +1,6 @@
-{:api-settings {:uri #immuconf/override "Specify the uri where the API is hosted"}
+{:api-settings {:uri #immuconf/override "Specify the uri where the API is hosted"
+                :lambda {:clj #immuconf/override "Specify the uri for the clj lambda function"
+                         :js #immuconf/override "Specify the uri for the js lambda function"}}
```

Comment:
Ya, I got this in before I added the python handler so I was going to have another PR for python uris. :)